pkg/dataplane/http/context.go

```diff
@@ -1476,3 +1490,30 @@ func (c *context) stop(reason string, timeout *time.Duration) error {
 
 	return nil
 }
+
+// parsing the mtime from a header of the form `__mtime_secs==1581605100 and __mtime_nsecs==498349956`
+func parseMtimeHeader(response *v3io.Response) (int, int) {
+	var mtimeSecs, mtimeNSecs int
+	var err error
+
+	mtimeHeader := string(response.HeaderPeek("X-v3io-transaction-verifier"))
+	for _, expression := range strings.Split(mtimeHeader, "and") {
+		mtimeParts := strings.Split(expression, "==")
+		mtimeType := strings.TrimSpace(mtimeParts[0])
+		if mtimeType == "__mtime_secs" {
+			mtimeSecsStr := strings.TrimSpace(mtimeParts[1])
```

Comment:
You're repeating the parse logic here. Please move to a small helper function that returns int, error